src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, isMarried, partnerName, sex } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age !== undefined && <p className="Person__age">I am {age}</p>}
+      {isMarried ? (
+        <p className="Person__partner">
+          {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
[CODE STYLE] According to the checklist, you should format the ternary operator operands correctly by moving each operand to a separate line. This will improve the readability of your code. The corrected code will look like this: 

jsx
{isMarried 
  ? (
    <p className="Person__partner">
      {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
    </p>
  ) 
  : (
    <p className="Person__partner">I am not married</p>
  )}
```